src/App.css

```
@@ -1,54 +1,130 @@
+h2,
+p,
+label /* Add other text elements as needed */ {
+  color: inherit; /* Inherit color from body by default */
+}
+
+a {
+  color: var(--link-color-light);
+  transition: color var(--transition-speed) ease;
+}
+a:hover {
+  color: darken(var(--link-color-light), 10%); /* Slightly darken on hover */
```

Biome (1.9.4)

[error] Unexpected unknown function: darken

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)